hvplot/converter.py

```diff
@@ -2123,8 +2123,9 @@ def chart(self, element, x, y, data=None):
 
         charts = []
         for c in y:
-            kdims, vdims = self._get_dimensions([x], [c])
-            chart = element(data, kdims, vdims).redim(**{c: self.value_label})
+            ydim = hv.Dimension(c, label=self.value_label)
+            kdims, vdims = self._get_dimensions([x], [ydim])
+            chart = element(data, kdims, vdims)
             charts.append((c, chart.relabel(**self._relabel).redim(**self._redim)))
```

Comment:
Maybe let's also skip `relabel` and `redim` if `self._relabel` and `self._redim` are empty respectively.